text/0000-rename-distributed-ctx.md

```diff
@@ -0,0 +1,59 @@
+# Rename distributed context into tags and add support for Baggage.
+
+**Status:** `proposed`
+
+Rename Distributed-context to a less confusing term and restrict its use for observability.
+ 
+ 
+## Motivation
+
+Context Propagation is generalized to carry observability and non-observability contexts. This
+includes SpanContext, Distributed-context, Authentication-context, etc. Distributed-context
+carries key-value pairs for observability. The name Distributed-context is
+very confusing. Hence it should be changed to TagMap (or other better suggestion). 
+That is the first motivation.
+ 
+The second motivation is the restrict use of Distributed-context to observability. As it is
+written currently it does not specifically call out for using it only for non-observability purpose.
+However, it can be used to support 'Baggage' functionality for non-observability purpose as it is
```

Comment:
how is "non-observability" defined? I would imagine here that if Baggage refers to the key-values pairs that carry with the trace, then those would contribute to observability since they help define the context of the request.

Reply:
non-observability should be defined as data that needs to propagate downstream along with the request for usage specific to application, testing, etc. It is not tied to whether the request is being traced/monitored or not.

Reply:
Got it! might be good to add that definition somewhere, unless there is a definition floating in the spec somewhere.